src/com/estafet/learning/US008/Order.java

```diff
@@ -0,0 +1,27 @@
+package com.estafet.learning.US008;
+
+public abstract class Order {
+    public static final String ORDER_NAME = "ORR";
+    public String orderNumber;
+    public String ClientDetails;
+    public String ListWithArticles;
+    public String TotalAmount;
+    private static final int invoice = 1;
```

Comment:
if this is constand, name should be in CAPITAL

Reply:
Ivo, please read this: https://www.theserverside.com/feature/Java-naming-conventions-explained#:~:text=For%20variables%2C%20the%20Java%20naming,a%20lower%20camel%20case%20syntax.
and change all variables according to the namin conventions in Java